test/jdk/java/lang/ProcessBuilder/UnblockSignals.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (c) 2022, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+import java.lang.Process;
+import java.lang.ProcessBuilder;
+import java.io.IOException;
+
+/*
+ * @test
+ * @summary Verify Signal mask is cleared by ProcessBuilder start
+ * @bug 8234262
+ * @run main/othervm UnblockSignals
+ * @run main/othervm -Xrs UnblockSignals
+ */
+public class UnblockSignals {
+    public static void main(String[] args)  throws IOException, InterruptedException {
+        // Check that SIGQUIT is not masked, in previous releases it was masked
+        final ProcessBuilder pb = new ProcessBuilder("sleep", "30").inheritIO();
```

Comment:
SIGQUITing the child may leave a core dump. You may want to wrap the sleep in a shell call with ulimit to prevent that, e.g. 

`new ProcessBuilder("sh", "-c", "ulimit -c 0 && sleep 30");`

Reply:
That doesn't work as intended; the `sh` is killed, not the sleep; and the core dump would be of `sh`.
The core dump of sleep will be small, I think it can be tolerated if it happens.